src/components/menu-surface/menu-surface.scss

```diff
@@ -1,7 +1,30 @@
 @import '../../style/variables';
-@import "@limetech/mdc-menu-surface/mdc-menu-surface";
+@import '@limetech/mdc-menu-surface/mdc-menu-surface';
+@import '@limetech/mdc-elevation/mdc-elevation';
 
 .mdc-menu-surface {
     width: 100%;
     max-height: 100%;
 }
+
+.dropdown--spinner {
+    padding-top: pxToRem(6);
+    text-align: center;
+}
+
+.dropdown--spinner,
+.dropdown--list {
+    background-color: white;
+    position: absolute;
+    margin-top: pxToRem(-1);
+
+    &:not(.display-full-list) {
+        max-height: pxToRem(250);
+        overflow-y: auto;
+    }
+}
+
+.empty-result-message {
+    color: var(--lime-light-grey, #{$lime-light-grey});
+    text-align: center;
+}
```

Comment:
Should these classes be part of the menu surface component? It feels like they are part of the picker or the spinner. Would it be possible to refactor these so they belong to the component that uses them? I hope that made sense :slightly_smiling_face:

Reply:
good catch, that might actually not be needed anymore because I'm setting it as inline css on the paragraph (in the picker) 

the problem is that html wise it's not part anymore of the picker, that's why it can't be in the css file for limel-picker